decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -219,8 +219,15 @@ def generate_localized_title(field = nil, skip_injection: false)
     end
 
     trait :deleted do
+      name { "" }
+      nickname { "" }
       email { "" }
+      delete_reason { "Fooo delete bar" }
```

Comment:
Let's add the typical reason used in other tests here:

```suggestion
      delete_reason { "I want to delete my account" }
```